src/app/index.ts

```diff
@@ -265,7 +274,13 @@ module.exports = yo.extend({
         // Copy project template files from project repository (currently only custom functions has its own separate repo)
         if (projectRepoBranchInfo.repo)
         {
-          git().clone(projectRepoBranchInfo.repo, this.destinationPath(), ['--branch', (projectRepoBranchInfo.branch) ? projectRepoBranchInfo.branch : 'master']);
+          git().clone(projectRepoBranchInfo.repo, this.destinationPath(), ['--branch', (projectRepoBranchInfo.branch) ? projectRepoBranchInfo.branch : 'master'], (err) => {
+            const gitFolder = this.destinationPath() + '/.git';
+            if (fs.existsSync(gitFolder)){
+              helperMethods.deleteFolderRecursively(gitFolder);
+            }
+            return err? reject(err): resolve();

```

Comment:
nit: uses spaces on both sides of `?` and `:`.
 `return err ? reject(err) : resolve();`

Reply:
Adam, I will make these changes since I have incorporated these changes into another change I am about to send out that calls into office-addin-manifest to modify the manifest guid and display name

Reply:
@TCourtneyOwen I think it would be better to merge this first, and then we can merge it into your manifest change branch.

Reply:
OK, that's fine, but I am going to make these changes as opposed to waiting on Thorsten